docs/blog.md

```diff
@@ -404,11 +404,11 @@ Now we had our backend polling logic and waku setup ready, lets import these fun
     }
 ```
 
-You will notice the code blocks above uses other state values like wakuNodeStatus, lastSynced these are metrics to show user the peer connection status and last received time of a waku message.
+You will notice the code blocks above uses other state values like `wakuNodeStatus` or `lastSynced` these are metrics to show the user the peer connection status and last time we received a message ove Waku.
 
 #### Wrap
 
-Although the problem that I solved here can be implmented many ways, having decentralized solutions makes your app fully decentralized. Waku can be applied in several other cases too.
+And it's a wrap. Some of you may yell at screen that this problem can be solved in other ways. Yes it is, but the goal over here is to show how Waku plays key role in DApps irrespective of the size of the application. Additionally, users of DApp's may look for complete decentralization, if it misses then companies may loose their customers. So, to keep fully decentralize setup for your DApp one of the important characteristic is decentralized communication. By integrating Waku, it enables decentralised communication features to your application without compromising security or privacy or may be scalability if you are integrating in large applications.
```

Comment:
@vpavlin Updated conclusion. Other changes looks fine. Let me know thoughts on the summary.